beacon-chain/casper/incentives.go

```diff
@@ -13,14 +13,20 @@ var log = logrus.WithField("prefix", "casper")
 // based on FFG incentive structure.
 func CalculateRewards(attestations []*pb.AggregatedAttestation, validators []*pb.ValidatorRecord, dynasty uint64, totalDeposit uint64) ([]*pb.ValidatorRecord, error) {
 	activeValidators := ActiveValidatorIndices(validators, dynasty)
-	attesterDeposits := GetAttestersTotalDeposit(attestations)
-
+	attesterDeposits := AttestersTotalDeposit(activeValidators)
```

Comment:
Is this actually the total deposit of the attesters for this cycle? Looks like this is just the size of the active validator set multiplied by the default balance size.

Reply:
+1

Reply:
It's the same thing right? every validator gets to vote once per cycle and you have to be active to vote

Reply:
if thats the case change the naming to `ValidatorsTotalDeposit()`

Reply:
Done

Reply:
`attesterDeposits` is the total amount that voted during the cycle. `totalDeposit` is the total number that could have voted during the cycle.

Right now, `attesterDeposits` looks like the total amount that could have voted.

Reply:
True. I'm getting the `totalParticipatedDeposits` using `blockVoteCache`

Found another bug. When we check for `justifiedBlock` we do:
```go
		if 3*blockVoteBalance >= 2*c.TotalDeposits() {
			if slot > justifiedSlot {
				justifiedSlot = slot
			}
```
it shouldn't be `2*c.TotalDeposits()` it should be the total balance of the validators in that slot committee 😅